motionplan/motionPlanner.go

```diff
@@ -509,21 +509,24 @@ func CheckPlan(
 		}
 
 		// get checkFrame's currentInputs
+		// *currently* it is guaranteed that a relative frame will constitute 100% of a solver frame's dof
```

Comment:
Why is this guaranteed? Wouldn't it be possible to have an arm on a base?
edit: hmm, taking a look at `solverFrame.go` is it because we removed `solveFrame` in favor of `solveFrameName`?

Reply:
https://github.com/viamrobotics/rdk/blob/59510a8f0d057007e789b0a23d38dc6e8fd89619/motionplan/solverFrame.go#L164